portfolio/src/main/webapp/script.js

```diff
@@ -190,8 +190,45 @@ function createNextButton(direction, isValid) {
   return nextButton;
 }
 
```

Comment:
A lot of this is code is very repetitive right now (createGameMarker vs. createMarker, createInteractiveMap vs createStaticMap).. I will definitely make a createMap helper function that the other map functions can use. Not sure about the markers yet